tests/unit/country.test.js

```diff
@@ -0,0 +1 @@
+const country = require("./cmds/country");
```

Comment:
Looks like this file path is incorrect. Tests are failing.

Reply:
hmm weird thats the only thing in the file. Could you help me with creating a test file. I am a bit confused

Reply:
Talked this through with Malik over Slack. He is going to work on a test script.